scripts/combine_control_files.py

```diff
@@ -0,0 +1,35 @@
+# Script to generate ext_index.json metadata file
+# that stores content of the control files and location of extension archives
+#  for all extensions in /extensions subdir.
+import json
+import os
+import sys
+from os.path import isdir, join
+
```

Comment:
All the logic should be inside `if __name__ == "__main__":` block (or in functions)

Reply:
ok